packages/entity-browser/package.json

```diff
@@ -4,10 +4,10 @@
   "description": "Lists entities and allows to sort and search them",
   "main": "dist/index.js",
   "scripts": {
-    "clean": "rimraf dist",
+    "clean": "../../node_modules/rimraf/bin.js dist",
     "compile": "babel-node ../../bin/compile",
```

Comment:
is this script necessary? if yes, where is the package name defined? compile script throws an error without a packagename provided